migrations/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM openjdk:8-jre-alpine
+
+RUN apk --no-cache add openssl
+
+WORKDIR /flyway
+
+ENV FLYWAY_VERSION 4.2.0
```

Comment:
flyway is saying that 5.2.4 is out and 6 is coming - any reason to use this older version?

Reply:
No good reason at all - so I attempted to upgrade and it was failing with coufounding syntax error, apparently in the flyway cmd file itself. There seemed to be some precedent for this, where flyway and alpine weren't playing nicely (https://github.com/flyway/flyway-docker/issues/4). 

But in the course of looking at that, I realised that there were already docker images with flyway pre-installed (not sure if they post-date our earlier flyway integration), so thought I'd try one of those instead. 

The basic version works great, and that's what I've changed this to. The alpine version crashes the JRE! Don't know if that's some variation of the original problem or something new. Alpine is largely used for being lightweight? OK to use the standard flyway image?